src/java/org/jivesoftware/openfire/plugin/util/cache/ClusteredCacheFactory.java

```diff
@@ -423,7 +423,7 @@ public void doClusterTask(final ClusterTask<?> task, final byte[] nodeID) {
             try {
                 final Future<T> future = hazelcast.getExecutorService(HAZELCAST_EXECUTOR_SERVICE_NAME).submitToMember(new CallableTask<>(task), member);
                 result = future.get(MAX_CLUSTER_EXECUTION_TIME, TimeUnit.SECONDS);
-                logger.debug("DistributedTask result: " + (result == null ? "null" : result));
+                logger.trace("DistributedTask result: " + (result == null ? "null" : result));
```

Comment:
As you're editing this line, would be good to
a) replace the String concatentation with a log template, and 
b) remove the superflous  null check

Reply:
done